### DISCUSSION TITLE
How to change the workspace on monitor?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [hyprwm/Hyprland#5427] [Q&A]
For example, `workspace, m1` will go to the first workspace on the external monitor, and `workspace, m2' to the second.

**Comment by 0xFMD** [comment]
could be setting [workspace rules](https://wiki.hyprland.org/Configuring/Workspace-Rules).
like:
```
workspace = {your monitor},{desired workspace}, default:true

```

**Reply by azizsayfiddin** [reply]
So that when I press alt+2, it should switch to the second workspace on the active monitor, even if the workspace with id 2 is on a different workspace.
